po/es.po

```diff
@@ -131,34 +131,34 @@ msgstr "Actual: %.1f °C"
 
 #, c-format
 msgid "Command line is empty."
-msgstr "La línea de comando està vacía."
+msgstr "La línea de órdenes está vacía."
 
 #, c-format
 msgid "Unable to execute command."
-msgstr "Imposible ejecturar el comando."
+msgstr "No se puede ejecutar la orden."
 
 #, c-format
 msgid "Command has exited with status code %d."
-msgstr "El comando terminó con el código de estado %d."
+msgstr "La orden terminó con el código de estado %d."
 
 #, c-format
 msgid "Command did not return valid numbers."
-msgstr "El comando no ha producido números validos"
+msgstr "La orden no ha devuelto números validos."
```

Comment:
Thanks for this excellent update! While you’re here, could you also fix that lingering typo in “validos” (missing accent mark)?

Reply:
Good catch! Fixxed in 43ec21a.

Thanks